### DISCUSSION TITLE
Alternative to compiler.removeConsole in next.config.js for turbopack

### DISCUSSION THREAD

**Discussion by jinsley8** [vercel/next.js#57555] [Help]
### Summary

I'm testing `next dev --turbo` in `v14.0.0` with pages directory but I get an error:  

```
- Unsupported Next.js configuration option(s) (next.config.js)
    To use Turbopack, remove the following configuration options:
        - compiler.removeConsole
```

Is there an alternative for turbopack to remove console logs in production?

  
This is what I'm using now:  

```
const nextConfig = {
    compiler: {
        removeConsole: process.env.NODE_ENV === 'production',
    },
}
```

**Comment by junglast** [comment]
Any progress on this? 😢

**Comment by eseQ** [comment]
Try to add it only for production config:
`if (process.env.NODE_ENV === 'production') nextConfig.compiler = { removeConsole: true };`

**Reply by waflessnet** [reply]
work  :rabbit:

**Comment by zigang93** [comment]
@leerob does turborepo gonna support `removeConsole` in future `14.1.1` version ?

**Comment by GrahamQuan** [comment]
Any update?
